queries.sql

```diff
@@ -72,4 +72,51 @@ GROUP BY neutered;
 select species, min(weight_kg) as minimum, max(weight_kg) as maximum from animals 
 GROUP BY species
 select species, avg(escape_attempts) from animals 
-where extract(YEAR FROM date_of_birth) BETWEEN 1990 AND 2000 GROUP BY species
\ No newline at end of file
+where extract(YEAR FROM date_of_birth) BETWEEN 1990 AND 2000 GROUP BY species
+
+/*Queries with JOINs*/
+-- What animals belong to Melody Pond?
+SELECT name, full_name 
+    FROM animals 
+    INNER JOIN owners 
+    ON animals.owner_id = owners.id 
+    WHERE full_name = 'Melody Pond';
+
+
+-- List of all animals that are pokemon (their type is Pokemon).
+SELECT a.name, s.name AS type
+    FROM animals a 
+    INNER JOIN species s 
+    ON a.species_id = s.id 
+    WHERE s.name = 'Pokemon';
```

Comment:
- [ ] Again as requested by the previous reviewer, please use the * to request all attributes. It will look like but not limited to this 👇 

```sql
   SELECT * FROM animals
   LEFT JOIN species
   ON animals.species_id = species.id
   WHERE species.name = 'Pokemon';
```

Reply:
Hello @BertrandConxy thank you for the review to my code but I will like to point out that these suggestions are not one of the requirements for this exercise. Why are guys restricting me to choose your own method; I think it should be a **suggestion** not something to stop the project approval.